keyboards/handwired/dactyl_manuform/6x6/keymaps/default_5_thumb/keymap.c

```diff
@@ -0,0 +1,47 @@
+// #include QMK_KEYBOARD_H
+#include QMK_KEYBOARD_H
+
+#define _QWERTY 0
+#define _LOWER 1
+#define _RAISE 2
```

Comment:
Should use an enum here instead
```suggestion
enum custom_layers {
    _QWERTY,
    _LOWER,
    _RAISE,
};
```

Reply:
done